### ISSUE TITLE
CH12 Unkonwn loss function: HuberLoss?

### ISSUE THREAD

**Issue by CoderZYQin** [ageron/handson-ml2#72]
<img width="857" alt="屏幕快照 2019-12-11 18 29 42" src="https://user-images.githubusercontent.com/19929654/70613982-987e7680-1c44-11ea-86f1-52ffd0f3f8a9.png">
<img width="1013" alt="屏幕快照 2019-12-11 18 29 51" src="https://user-images.githubusercontent.com/19929654/70613986-9c11fd80-1c44-11ea-8670-0e176961ee47.png">

**Comment by ageron** [comment]
Hi @CoderZYQin ,
Unfortunately, this seems to be a bug in TensorFlow. There was a similar bug earlier this year, so I filed the issue [#25938](https://github.com/tensorflow/tensorflow/issues/25938), and it was resolved after a while.
Could you please create a minimalistic code example that highlights the problem, and open a [new TensorFlow issue](https://github.com/tensorflow/tensorflow/issues/new?template=00-bug-performance-issue.md)? Thanks a lot!
